MOBILE_API.xml

```diff
@@ -4685,6 +4686,21 @@
         </param>
     </struct>
 
+    <struct name="SeatStatus" since="7.1">
+      <description>Describes the status of a parameter of seat.</description>
+      <param name="seatLocation" type="SeatLocation" mandatory="true"/>
+      <param name="conditionActive" type="Boolean"  mandatory="true"/>
```

Comment:
```suggestion
      <param name="conditionActive" type="Boolean" mandatory="true"/>
```

Reply:
@jacobkeeler Done in 476cb36e4f08c8392a247bcdbf6e6c1541b76a9b